dilute.py

```diff
@@ -0,0 +1,68 @@
+import sys
```

Comment:
Can you add some comments to help us understand the carious parts of the code?

Also, I think it would make more sense if this is inside the `build` directory.

Reply:
Or see my comment about folding all of this into `mech_converter.py` :)

Reply:
But comments and removal of all those blank lines is good